### PR TITLE
Add 'ErrorDlrNotFound' to errors

### LINKED ISSUE #51: implement ErrorDlrNotFound
Implement ErrorDlrNotFound

https://github.com/octoposprime/op-be-graphql/blob/3aa8f98467375ad91a80c02f50e7200ae4e2c7c7/pkg/presentation/controller/graphql/resolver/dlr.resolvers.go#L119

to 

https://github.com/octoposprime/op-be-graphql/blob/3aa8f98467375ad91a80c02f50e7200ae4e2c7c7/internal/domain/model/object/error.go#L18C1-L21C2

add necessarry fields to 	
[smodel "github.com/octoposprime/op-be-shared/pkg/model"](https://github.com/octoposprime/op-be-shared/blob/phase1/pkg/model/error.go)

### PR DESCRIPTION
**Pull request by mervegonc**
Closes #51
Add 'ErrorDlrNotFound' to errors

### CODE REVIEW

File: internal/domain/model/object/error.go [hunk 1]
```diff
@@ -14,7 +14,7 @@ var ERRORS []error = []error{
 
 const (
 	ErrUser string = "user"
-	ErrDlr  string = "dlr"
+	ErrDlr  string = "dlr",
```
Comment (Sddilora) [inline internal/domain/model/object/error.go]: "Comma unexpected." :') 
https://github.com/octoposprime/op-be-user/blob/be5ffaaaef4aac058f16bef029f055c64fcb12b5/internal/domain/model/object/error.go#L36-L47

**Review by Sddilora — CHANGES_REQUESTED** [review]
"But the return value is correct this time." 👍🏻

**Comment by Sddilora** [comment]
D'accord
S'il vous plaît
Corrigez
Le
Titre
De
La
Demande de tirage (pull request)

File: internal/domain/model/object/error.go (hunk 1 above)
Comment (mervegonc) [reply internal/domain/model/object/error.go:17]: finally, thanks to you <3

Comment (mervegonc) [reply internal/domain/model/object/error.go]: i fixed my syntax err, ı guess :|

**Comment by mervegonc** [comment]
> "But the return value is correct this time." 👍🏻

thanks to you <3

**Comment by mervegonc** [comment]
> D'accord S'il vous plaît Corrigez Le Titre De La Demande de tirage (pull request)

Do you comment when you are bored,
in french?

**Comment by Sddilora** [comment]
Our title needs to describe what we have done in this PR. "dlr add err" might refer to any error on DLR, so it could mean several things a month later, leading to potential trouble in managing our previous processes. We need to choose a more descriptive name, such as "Add 'ErrorDlrNotFound' to errors," so that we can understand what we have merged in this PR not only now but also a month or even a year later. Thank you for considering this change.

**Comment by Sddilora** [comment]
> > D'accord S'il vous plaît Corrigez Le Titre De La Demande de tirage (pull request)
> 
> Do you comment when you are bored, in french?

No, I was not bored. I'm sorry, but I got frustrated because we've discussed this problem many times and we're still encountering it. It seems like we're not giving our tasks the attention they deserve, and it puts a lot of pressure on our leads for small matters.

**Review by Sddilora — APPROVED** [review]

**Review by Feyzanrs — APPROVED** [review]

**Pull request merged**